lib/benchmark.rb

```diff
@@ -219,6 +219,9 @@ def bm(label_width = 0, *labels, &blk) # :yield: report
   # Because #bmbm takes two passes through the tests, it can
   # calculate the required label width.
   #
+  # Also #bmdm take +width+ argument. This argument is a initial value
+  # for the label offset used in formatting;
```

Comment:
This argument is legacy and it's better to not provide it as the method can compute it itself.
For `#bm`, it it useful though.